### ISSUE TITLE
low accuracy and bad segmentation 

### ISSUE THREAD

**Issue by zdd0819** [divamgupta/image-segmentation-keras#68]
![img-4025](https://user-images.githubusercontent.com/39216094/59270173-c3adec00-8c48-11e9-8685-0c4d36ecade8.jpg)
![img-4743](https://user-images.githubusercontent.com/39216094/59270174-c3adec00-8c48-11e9-958c-8d98d56fdd91.jpg)
![img-2332](https://user-images.githubusercontent.com/39216094/59270175-c3adec00-8c48-11e9-80c2-03aea6b3c445.jpg)
![img-3038](https://user-images.githubusercontent.com/39216094/59270177-c3adec00-8c48-11e9-8d26-cd9d0d612e83.jpg)

(data set is from SUNRGBD,and the label is 37)
when i finish the training, i just get the 40% accuracy, so how do i improve it? 

and when i predicted my test img, i got the following picture, do u know what's wrong here?

many thanks about that!

**Comment by divamgupta** [comment]
Could you give more details of the model used?

Maybe try finetuning from pretrained segmentation model
